hivemq-edge/src/frontend/src/api/hooks/useProtocolAdapters/useGetProtocolAdapter.tsx

```diff
@@ -6,8 +6,11 @@ import { QUERY_KEYS } from '@/api/utils.ts'
 export const useGetProtocolAdapter = (id: string) => {
   const appClient = useHttpClient()
 
-  return useQuery<Adapter>([QUERY_KEYS.ADAPTERS, id], async () => {
-    const adapter = await appClient.protocolAdapters.getAdapter(id)
-    return adapter
+  return useQuery<Adapter>({
+    queryKey: [QUERY_KEYS.ADAPTERS, id],
+    queryFn: async () => {
+      const adapter = await appClient.protocolAdapters.getAdapter(id)
+      return adapter
```

Comment:
suggestion(non-blocking): can be returned directly and aligns with other implementation within the PR
```suggestion
      return await appClient.protocolAdapters.getAdapter(id)
```